persistent/Database/Persist/TH.hs

```diff
@@ -141,6 +141,7 @@ import Database.Persist.ImplicitIdDef.Internal
 conp :: Name -> [Pat] -> Pat
 conp name pats = ConP name [] pats
 #else
+conp :: Name -> [Pat] -> Pat
```

Comment:
Fixes an annoying warning.
Unrelated but simple it's simple enough so I just keep it.
I could revert if you prefer.